stories/assets/js/accordion.js

```diff
@@ -1,35 +1,55 @@
 /* accordion JS start custom */
-export function accordion() {
+export function accordion(addclasses, siblingstag, accordiondiv, find) {
```

Comment:
@Shubham-Sharma-77 Wasn't this supposed to be done using `data` attributes on the html and then capturing that in this function based on the true or false value?